docs/DeveloperGuide.md

```diff
@@ -534,16 +529,16 @@ The following activity diagram summarizes what happens when a user executes the
 <div style="page-break-after: always;"></div>
 
 #### Design considerations
-**Aspect: Deletes assignment of a person in current displayed list or for any person in storage:**
+**Aspect: Deletes assignment of a person in currently displayed list or for any person in storage:**
 
-* **Alternative 1:** (current choice) Deletes assignment of a person in current displayed list
-    * Pros: Allows for a safer delete of assignments
-    * Cons: User has to carry out `list` command first if required person is not in the current displayed list
+* **Alternative 1:** (current choice) Deletes assignment of a person in the currently displayed list.
+    * Pros: Allows for a safer delete of assignments.
+    * Cons: User has to carry out `list` command first if required person is not in the current displayed list.
 
-* **Alternative 2:** Deletes assignment of any person with that name and assignment
-    * Pros: Allows user to delete assignment of a person without the need of additional commands
-    * Cons: User may not be certain about which person's assignment to delete if several of them has completed assignment
-    and likely to remember the wrong person name if the current person displayed list is not shown
+* **Alternative 2:** Deletes assignment of any person with that name and assignment.
```

Comment:
I think you can say "Deletes assignment of the specified person, by inputting their name with the command"